README.md

```diff
@@ -18,18 +18,15 @@ There is also a third NuGet package available: [AssertNet.Core](https://www.nuge
 Grab one of the available version mentioned in the __Downloads__ section above.  
 Add the following line to the test files where you intend to use _AssertNet_ depending on your version:  
 ```cs
-using static AssertNet.Assertions;
-```  
-For Moq:  
-```cs
-using static AssertNet.Moq.Assertions;
+using static AssertNet.AssertionBuilder;
```

Comment:
this also now requires `using AssertNet;` in order for the extension methods to be loaded.